Lesson_4/homework_04.py

```diff
@@ -69,7 +72,9 @@
 """ Виведіть четверте речення з adwentures_of_tom_sawer_sentences.
 Перетворіть рядок у нижній регістр.
 """
-print(adwentures_of_tom_sawer[16:21])
+
+four_line = lower_cases[431:536]
```

Comment:
The more clear and flexible decision will be - split your string with ". " identifier and then get  3rd sentence from list. But you solution is ok.